### ISSUE TITLE
Scroll after lightly tapping, content should not be selected

### ISSUE THREAD

**Issue by swanfly** [ajaxorg/ace#5500]
### Describe the bug

在colum1位置轻点，然后向下滚动，colum1下部的内容不应该被选中。但是选中了。
Tap on the position of column1 and scroll down, the content below column1 should not be selected. But it was selected
<img width="594" alt="image" src="https://github.com/ajaxorg/ace/assets/5317370/6bcdfc2e-09c7-4940-be71-496b5de0ff1f">

### Expected Behavior

Scroll after lightly tapping, content should not be selected

### Current Behavior

Tap to scroll, but the content is selected

### Reproduction Steps

This website below uses ACE, and when I lightly clicked and scrolled down, the content was selected. I don't want to be selected, the experience is very bad.
https://markdownlivepreview.com

https://github.com/ajaxorg/ace/assets/5317370/4ca066a3-b73b-48de-ae1a-33220e84bd78

### Ace Version / Browser / OS / Keyboard layout

Safari  17.4

**Comment by andredcoliveira** [comment]
Is this something you can reproduce in https://ace.c9.io/build/kitchen-sink.html?

Also, are you on a desktop, mobile, or other? I presume you wouldn't be able to light tap without a touch screen

**Comment by swanfly** [comment]
> Is this something you can reproduce in https://ace.c9.io/build/kitchen-sink.html?
> 
> Also, are you on a desktop, mobile, or other? I presume you wouldn't be able to light tap without a touch screen

Yes, it does reproduce in  https://ace.c9.io/build/kitchen-sink.html
I used it on MacBook Pro, Mac OS Sonoma 14.4.
Chrome browser is completely normal, but Safari browser only has this issue.


https://github.com/ajaxorg/ace/assets/5317370/6bd5a8be-4890-47cc-8072-1f01fbd2cffa

**Comment by InspiredGuy** [comment]
What click and scroll settings do you have on your macbook? I cannot reproduce it in Safari on my macbook, both with one-tap-click turned on and off.

**Comment by swanfly** [comment]
> What click and scroll settings do you have on your macbook? I cannot reproduce it in Safari on my macbook, both with one-tap-click turned on and off.

Click on the text, swipe up and down, sometimes it will be selected.

**Comment by nightwing** [comment]
It is being selected when clicking near cursor and anchor of the selection, perhaps we should add teardrop handlers at the start and end to make it clearer that clicking there, is going to modify selection instead of scrolling.
